<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ShortBreakButton,#LongBreakButton,#PomodoroButton {
	border: 1px solid #8f8f91;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #FBFBFB, stop: 1 #E0E0E0);
	min-width: 80px;
}
#ShortBreakButton:hover,#LongBreakButton:hover,#PomodoroButton:hover {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #f6f7fa, stop: 1 #CCC);
	/*background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #EDF4F8, stop: 1 #C2D1DE);*/
}
#PomodoroButton {
	border-top-left-radius: 6px;
	border-bottom-left-radius: 6px;
}
#ShortBreakButton {
	border-left: 0px;
}
#LongBreakButton {
	border-top-right-radius: 6px;
	border-bottom-right-radius: 6px;
	border-left: 0px;
}
#StartButton {
	border: 1px solid #8f8f91;
	border-radius:5px;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #4FE455, stop: 1 #12A719);
	color:white;
	min-width: 180px;
	height:60px;
	font-size:17px/20px;
}
#StartButton:hover {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #3FF047, stop: 1 #15B91D);
}
#AddTaskEdit {
	height:20px;
	margin:0;
	padding:0;
	outline:none;
}
#AddTaskEdit:hover {
        /*box-shadow: 0 0 5px rgba(81, 203, 238, 1);*/
}
#AddTaskButton {
	color:#FFF;
	border: 1px solid #8f8f91;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #4FE455, stop: 1 #12A719);
	/*border-left: 0px;*/
	border-top-right-radius: 6px;
	border-bottom-right-radius: 6px;
	min-width: 80px;
	height:20px;
}
QPushButton {
	height:25px;
}
QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
 }
QPushButton:flat {
	border: none; /* no border for a flat push button */
 }
QPushButton:default {
	border-color: navy; /* make the default button prominent */
}
#SummaryWidget {
	background-color:#F6F6F6;
	border-top:1px solid #CCC;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="ButtonsGrid">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="PomodoroButton">
       <property name="font">
        <font>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>pomodoro</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="ShortBreakButton">
       <property name="text">
        <string>short break</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="LongBreakButton">
       <property name="text">
        <string>long break</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>661</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="StartGrid">
     <item row="0" column="1">
      <widget class="QPushButton" name="StartButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>start pomodoro</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="LeftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <spacer name="RightSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="SummaryWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>661</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QTabWidget" name="SummaryTabs">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>661</width>
       <height>291</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane { /* The tab widget frame */
     /*border-top: 2px solid #C2C7CB;*/
	/*background-color:#F6F6F6;*/
	border:0;
     position: absolute;
     top: -0.5em;
 }

QTabWidget::tab-bar {
     alignment: center;
 }
  QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
     /*border: 2px solid #C4C4C3;
     border-bottom-color: #C2C7CB; /* same as the pane color */
     /*border-top-left-radius: 4px;
     border-top-right-radius: 4px;*/
	background-color:#F6F6F6;
     min-width: 12ex;
     /*padding: 2px;*/
	padding:0;
	color: #AAA;
	font-size: 16px;
	font-family: 'PT Sans';
	/*font-weight:bold;*/
 }
QTabBar::tab:selected {
	color: #111;
	font-size: 16px;
	font-family: 'PT Sans';
	font-weight: bold;
}
QTabBar::tab:selected, QTabBar::tab:hover {
     /*background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                 stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);*/
 }
QTabBar::tab:hover {
	color:#111;
}
 QTabBar::tab:selected {
     /*border-color: #9B9B9B;
     border-bottom-color: #C2C7CB; /* same as pane color */
 }</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="TodoTab">
      <attribute name="title">
       <string>To-Do Today</string>
      </attribute>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>371</width>
         <height>46</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="AddTaskEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="AddTaskButton">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>+ add task</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>54</y>
         <width>371</width>
         <height>16</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="TasksVertical">
        <property name="spacing">
         <number>2</number>
        </property>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="HistoryTab">
      <attribute name="title">
       <string>History</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="StatsTab">
      <attribute name="title">
       <string>Stats</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
